rules/windows/process_creation/proc_creation_win_python_pty_spawn.yml

```diff
@@ -9,25 +9,26 @@ references:
     - https://www.volexity.com/blog/2022/06/02/zero-day-exploitation-of-atlassian-confluence/
 author: Nextron Systems
 date: 2022/06/03
+modified: 2023/02/03
```

Comment:
```suggestion
```
we shouldn't change the modified date for cosmetic changes

Reply:
@qasimqlf please update this